### ISSUE TITLE
Error No such file or directory @ rb_file_s_size

### ISSUE THREAD

**Issue by dellagustin** [jekyll-octopod/jekyll-octopod#63]
Hi All,

I have just started with `jekyll-octopod`, and I am having this issue during serve when hosting my episodes out of the `_episode` folder in a separate hosting.
`download_url` is set in `_config.yml`, yet, because I deleted the folder `_episodes` (or simply renamed the audio file), I am getting the following error:

```
 Liquid Exception: No such file or directory @ rb_file_s_size - ./episodes/episode0.mp3 in /_layouts/post.html
       Error: No such file or directory @ rb_file_s_size - ./episodes/episode0.mp3
       Error: Run jekyll build --trace for more information.
```

any tips on what I could be doing wrong? Or does it look like a bug?

I think I can live with it, as I could store the files there but ignore it in `.gitignore`, although I did not try that yet.

**Comment by dellagustin** [comment]
So I realized that this is due to the file size liquid filter https://jekyll-octopod.github.io/liquid_filters
but I don't understand what kind of workflow am I supposed to use when building the site for a podcast that hosts the audio in a different domain.

I can only imagine I have to hardcode the filesize in the post's front matter, but is there an official way to do it?

I am thinking about something like:
```
audio:
  mp3: untitled_20200319/untitled.mp3
filesize:
  mp3: 123456
```

And then adapt every usage of file_size replacing it by filters that will fetch the hardcoded file size.

**Comment by haslinger** [comment]
I will have a look into this, not sure, if very soon (a bit stressy right now over here).
You are probably right with your assumptions, I would currently assume, it's a bug.
Maybe we first added the external hosting feature, then the filesize into the rss-feed and it's broken right now. I don't remember. And the fallback you suggested would definitely be a way to go.

**Comment by dellagustin** [comment]
Thanks a lot for the quick response!
I will proceed with my approach.
Stay safe.

**Comment by haslinger** [comment]
* fixed with 7f9b751 or the gem version 0.9.14
* There is no reason for you to update, as you mitigated the issue yourself
